configs/sot/mixformer/README.md

```diff
@@ -30,18 +30,18 @@ We now provide the evaluation results using model weights released in [official
 
 ### Lasot
 
-|  Method   | Style | Inf time(fps) | Success | Norm precision | Precision |                 Config                  |
-| :-------: | :---: | :-----------: | :-----: | :------------: | :-------: | :-------------------------------------: |
-| MixFormer |   -   |       -       |  69.2   |      78.8      |   74.3    | [config](./mixformer_cvt_500e_lasot.py) |
+|  Method   | Style | Inf time(fps) | Success | Norm precision | Precision |                 Config                  |                                                        Download                                                        |
+| :-------: | :---: | :-----------: | :-----: | :------------: | :-------: | :-------------------------------------: | :--------------------------------------------------------------------------------------------------------------------: |
+| MixFormer |   -   |       -       |  69.2   |      78.8      |   74.3    | [config](./mixformer_cvt_500e_lasot.py) | [model](https://download.openmmlab.com/mmtracking/sot/mixformer/mixformer_cvt_500e_lasot/mixformer_cvt_500e_lasot.pth) |
 
 ### TrackingNet
 
-|  Method   | Style | Inf time(fps) | Success | Norm precision | Precision |                    Config                     |
-| :-------: | :---: | :-----------: | :-----: | :------------: | :-------: | :-------------------------------------------: |
-| MixFormer |   -   |       -       |  81.1   |      86.6      |   80.0    | [config](./mixformer_cvt_500e_trackingnet.py) |
+|  Method   | Style | Inf time(fps) | Success | Norm precision | Precision |                    Config                     |                                                        Download                                                        |
```

Comment:
Please update the model-index.yml like https://github.com/open-mmlab/mmtracking/blob/master/model-index.yml#L8